doc/CODING.md

```diff
@@ -104,6 +109,7 @@ Use `\copydoc`, `\copybrief` and `\copydetails` intensively.
 
 Files should start with:
 
+\verbatim
```

Comment:
Lets give the example to a separate file not parsed or lets use a newline between code block and doxygencommands so that github can at least make a code block.